resources/prosody-plugins/mod_filter_iq_jibri.lua

```diff
@@ -0,0 +1,26 @@
+local st = require "util.stanza";
+
+-- filters jibri iq in case of requested from jwt authenticated session that
+-- has features in the user context, but without feature for recording
+module:hook("pre-iq/full", function(event)
+    local stanza = event.stanza;
+    if stanza.name == "iq" then
+        local jibri = stanza:get_child('jibri', 'http://jitsi.org/protocol/jibri');
+        if jibri then
+            local session = event.origin;
+            local token = session.auth_token;
+
+            if jibri.attr.action == 'start'
+                and jibri.attr.recording_mode == 'file'
+                and (token == nil
+                    or session.jitsi_meet_context_user == nil
+                    or session.jitsi_meet_context_user["features"] == nil
+                    or session.jitsi_meet_context_user["features"]["recording"] ~= "true") then
```

Comment:
will there be an issue if `session.jitsi_meet_context_user["features"]["recording"]` is not there at all? or will that evaluate to `~= true` fine?